src/_component/open-source/open-source.scss

```diff
@@ -0,0 +1,294 @@
+.open-source__titluMult,
+.open-source__secondTitle {
+  font-size: 22px;
+  line-height: 30px;
+  font-weight: 600;
+  color: #707070;
+}
+
+// .listaGit {
+//   border: 2px solid #707070;
+//   list-style: none;
+// }
+
+// .eleGit {
+//   display: grid;
+//   grid-template-columns: auto 1fr auto;
+//  }
+
+// .cacaunu {
+//   color: #808080;
+//   font-size: 14px;
+//   font-weight: medium;
```

Comment:
you can delete all this commented code if you won't need it anymore.

Reply:
Oh god haha. I deleted that, I do not know why it is still visible 🙈